src/CommunityToolkit.Maui.MediaElement/readme.txt

```diff
@@ -0,0 +1,27 @@
+﻿.NET MAUI Community Toolkit
+
+## Initializing
+
+In order to use the .NET MAUI Community Toolkit MediaElement you need to call the extension method in your `MauiProgram.cs` file as follows:
+
+```csharp
+using CommunityToolkit.Maui.MediaElement;
+
+public static MauiApp CreateMauiApp()
+{
+    var builder = MauiApp.CreateBuilder();
+
+    // Initialise the toolkit
```

Comment:
can you update the readme following our updated one?
You can find it [here](https://github.com/CommunityToolkit/Maui#getting-started)